src/fakeData/facilities.ts

```diff
@@ -0,0 +1,58 @@
+import { HealthcareProfessional, Locale, LocaleName, Degree, 
+    Specialty, SpecialtyName, SpokenLanguage, Insurance, Facility } from '../typeDefs/dbSchema'
+
+export const fakeFacilities = () => {
+    // const name : LocaleName = {
```

Comment:
You could delete these comments since we have a different method to create a healthcare prof